mon-pix/app/controllers/assessments/challenge.js

```diff
@@ -28,11 +29,25 @@ export default class ChallengeController extends Controller {
   get pageTitle() {
     const stepNumber = progressInAssessment.getCurrentStepNumber(this.model.assessment, this.model.currentChallengeNumber);
     const totalChallengeNumber = progressInAssessment.getMaxStepsNumber(this.model.assessment);
-    const title = this.model.challenge.focused ? focusedPageTitle : this.challengeTitle;
+
+    let title;
+
+    if (this.model.challenge.focused) {
+      title = this._focusedPageTitle();
+    } else {
+      title = this.challengeTitle;
+    }
```

Comment:
```suggestion
    const title = this.model.challenge.focused ? this._focusedPageTitle() : this.challengeTitle;
```

C'est une pure préférence personnel, à voir si c'est partagé ou non.
Non bloquant.